content/articles/export-controls-us-china-chips.md

```diff
@@ -0,0 +1,47 @@
+---
+title: New US export regulations will deprive China of equipment for manufacturing high-end chips 
+description: 
```

Comment:
The title should be shorter
The description must not be empty
The filename better be something like: `2022-us-export-controls-on-china-chips` (this can be done by renaming the file while in edit mode)